common/postAcceptanceForm.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { ReactNode } from 'react';
+import { makeCheckbox, makeShortText, makeDropdown, makeLongText, makeSection } from './questions';
+import { QuestionDefinition, QuestionSection } from './types';
+
+export const PostAcceptanceForm: Array<QuestionSection | QuestionDefinition> = [
+  makeSection(
+    <>Acceptance Form</>,
+    <div>
+      <p>
+        Congrats on being accepted to HackBeanpot! We're so excited to see you soon. Please answer
+        the following questions to help us finalize logistics of making the event as great as it can
+        be for our attendees!
+      </p>
+      *NOTE: We will only be able to guarantee swag for the first 200 hackers who complete this form
+      and qualify for domestic shipping. Thanks for your understanding.
+    </div>
+  ),
+  makeShortText(
+    'email',
+    <div>
+      <p>Email address</p>
+      Please use the email address you applied to HackBeanpot with - this should be the one where
+      you received your acceptance email!
+    </div>,
+    true,
+    'Email address'
+  ),
+  makeShortText('name', 'Full name', true, 'First Last'),
```

Comment:
we already have their full name theoretically (though it looks like some hackers only added their first name). so we can probably delete this question.

if we want their full separate first and last names, separate this out into two separate questions, one for first name, the other for last name.